Userland/Libraries/LibJS/Runtime/GlobalObject.cpp

```diff
@@ -374,7 +374,7 @@ JS_DEFINE_NATIVE_FUNCTION(GlobalObject::parse_float)
     if (vm.argument(0).is_number())
         return vm.argument(0);
     auto input_string = TRY(vm.argument(0).to_string(global_object));
-    auto trimmed_string = input_string.trim_whitespace(TrimMode::Left);
+    auto trimmed_string = MUST(trim_string(global_object, js_string(vm, input_string), AK::TrimMode::Left));
```

Comment:
the `AK::` qualifier seems to be in excess?

Reply:
yep, my bad.

Reply:
Doesn't appear to be fixed :^)

Reply:
oops, I fixed it only in parseInt, fixing